mock/cpp_mock_scenarios/src/collision/spawn_with_offset.cpp

```diff
@@ -30,10 +29,7 @@ class SpawnWithOffsetScenario : public cpp_mock_scenarios::CppScenarioNode
 {
 public:
   explicit SpawnWithOffsetScenario(const rclcpp::NodeOptions & option)
-  : cpp_mock_scenarios::CppScenarioNode(
-      "spawn_with_offset",
-      ament_index_cpp::get_package_share_directory("cargo_delivery") + "/maps/kashiwa",
```

Comment:
.osm file should be exactly fixed in scenario.
So, please specify it in constructor.